index.js

```diff
@@ -164,23 +164,30 @@ class Replicate {
       progress(prediction);
     }
 
-    prediction = await this.wait(
-      prediction,
-      wait || {},
-      async (updatedPrediction) => {
-        // Call progress callback with the updated prediction object
-        if (progress) {
-          progress(updatedPrediction);
-        }
+    if (strict && !identifier.version) {
+      // Language models only support streaming at the moment.
```

Comment:
Clarify potential ambiguity in use of "only" here

```suggestion
      // Currently, only language models support streaming
```